README.md

```diff
@@ -13,6 +13,7 @@ installing the `ocne-catalog` chart.
 
 ### Supported for Oracle Cloud Native Environment 2.0
 
+=======
```

Comment:
Is this line left over from resolving a merge conflict?  I think it should be removed.

Reply:
Removed.